### ISSUE TITLE
introduces transparency where there is none making many pages almost unreadaable.

### ISSUE THREAD

**Issue by Lew-Rockwell-Fan** [M-Reimer/togglewebsitecolors#14]
This introduces transparency to elements of many pages that should be opaque, so that I often have multiple layers of text superimposed.  I'd prefer it only changed the color.  Alternately, I wouldn't mind if it removed all transparency or at least had that as an option,  like a 3-state toggle.  The attached images from mewe.com  are a good example.
![Screenshot_1d](https://github.com/M-Reimer/togglewebsitecolors/assets/16430650/07c9ee1e-b611-4230-869a-b937b52757a8)
![Screenshot_1e](https://github.com/M-Reimer/togglewebsitecolors/assets/16430650/9a7b12fb-143c-4785-b830-b1a35b0b383f)

**Comment by Lew-Rockwell-Fan** [comment]
2 more examples from the same site.
![2-without](https://github.com/M-Reimer/togglewebsitecolors/assets/16430650/33f9a96f-e88c-4612-bfa6-76eeb736125d)
![2-with](https://github.com/M-Reimer/togglewebsitecolors/assets/16430650/5db63018-0724-4d4b-a8fe-139c9c41d017)

**Comment by M-Reimer** [comment]
The problem is that all I can do is force a CSS to pages. And CSS doesn't know about the current style done by page CSS.

So I can either force background transparent or opaque. There won't be a "one fits all". And as your report is the first one like this for years, I guess forcing transparent is better in most cases.

Maybe you want to have a look at my other Add-on which works by asking the Firefox backend to enforce colors. It doesn't use CSS:

https://addons.mozilla.org/en-US/firefox/addon/toggleglobalcolors/

**Comment by Lew-Rockwell-Fan** [comment]
Indeed.  On first trial, that works better for me, at least on that site  - and also here on github as a matter of fact. Thanks.

**Comment by M-Reimer** [comment]
It uses a Firefox backend feature and the Firefox backend can know which site parts are transparent and which are not. Not possible with simple CSS overrides.

The "toggle by tab" Add-on was mostly meant to be used in the rare case where the colors of a website are totally unacceptable. So it was mostly optimized to be used "on demand". The "automatically enable for each new tab" was added later as people asked for it, but it is disabled by default for a reason.

I'll keep this bug open until I had a look at some of the "Global dark mode" Add-ons, just to see how they solved this issue. But likely I'll just keep with the current state as your report is the first after 7 years and so probably the current behavior works for most users.

**Comment by M-Reimer** [comment]
OK, it actually was mentioned before: https://github.com/M-Reimer/togglewebsitecolors/issues/5#issuecomment-513288211
But I'll take this as an opportunity to check for a potential solution, again. If the result is the same as in mentioned issue, I'll close here, too.
